ddp-workspace/projects/ddp-testboston/src/app/components/app/app.component.ts

```diff
@@ -31,6 +31,23 @@ export class AppComponent implements OnInit, OnDestroy {
     this.anchor.removeAll();
   }
 
+  public scrollToStep(event: MouseEvent): void {
+    const element = event.target as HTMLElement;
+    if (element.className.includes('scrollable')) {
+      const id = element.classList[element.classList.length - 1];
+      const step = document.getElementById(id);
```

Comment:
Unfortunately, it doesn't look like we can use the `id` attribute. If I put something like this in the backend:
```
bodyTemplate: {
   ...
   "template": "... <div id="step1"></div> ..."
}
```

Then what gets rendered on Angular side is this:
```
<div></div>
```

In other words, the `id` attribute gets stripped out by Angular. Can we configure Angular NOT to strip it out? Maybe we match on a CSS `class` instead? (something like `step1-marker` for `step1`?)

Reply:
This is an interesting one, thanks, will fix it.

Reply:
Fixed. Target section sould have `class="anchor-step1"` css class.

Reply:
Sounds good! I'll put that in the activity configuration.